monoio-http/src/h2/proto/streams/streams.rs

```diff
@@ -106,7 +107,7 @@ struct Actions {
 /// Contains the buffer of frames to be written to the wire.
 #[derive(Debug)]
 struct SendBuffer<B> {
-    inner: Mutex<Buffer<Frame<B>>>,
+    inner: Buffer<Frame<B>>,
 }
```

Comment:
Put the UnsafeCell at the place of Mutex is better. It’s good for later cherry pick changes.